src/core/urdb.jl

```diff
@@ -313,12 +313,12 @@ end
 
 
 """
-    parse_demand_rates(d::Dict, year::Int; bigM=1.0e8)
+    parse_demand_rates(d::Dict, year::Int; bigM=1.0e8, time_steps_per_hour::Int)
 
 Parse monthly ("flat") and TOU demand rates
     can modify URDB dict when there is inconsistent numbers of tiers in rate structures
 """
-function parse_demand_rates(d::Dict, year::Int; bigM=1.0e8)
+function parse_demand_rates(d::Dict, year::Int; bigM=1.0e8, time_steps_per_hour::Int)
```

Comment:
@hdunham  I think it's okay to leave as-is since these get called within a function that already sets the default to 1 (`function URDBrate(urdb_response::Dict, year::Int=2019; time_steps_per_hour=1)`)